assets/js/store.js

```diff
@@ -62,12 +62,26 @@ Store.prototype.has = function (key/*: string*/)/*: bool*/ {
 
 Store.prototype.able = function ()/*: bool*/ {
     try {
+        throwLocalStorageNotAccessibleError();
```

Comment:
Perfect! It seems `SecurityError` is the name of the exception in FF. Looking for something similar in Chrome!